<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>triggerDialog</class>
 <widget class="QDialog" name="triggerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extended Triggers</string>
  </property>
  <widget class="QTreeWidget" name="triggerListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>171</width>
     <height>541</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QTreeWidget" name="triggerMembersWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>591</width>
     <height>521</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="triggersNewCategoryButton">
      <property name="toolTip">
       <string>Add New Trigger Category</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="triggersNewTriggerButton">
      <property name="toolTip">
       <string>Add New Trigger</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="triggersNewTriggerComment">
      <property name="toolTip">
       <string>Add New Trigger Comment</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="triggersNewEventButton">
      <property name="toolTip">
       <string>New Event</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="triggersNewConditionButton">
      <property name="toolTip">
       <string>New Condition</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="triggersNewActionButton">
      <property name="toolTip">
       <string>New Action</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
